ldaptor/test/test_server.py

```diff
@@ -683,13 +681,7 @@ def test_passwordModify_notBound(self):
                 id=2).toWire())
 
     def test_passwordModify_simple(self):
-        data = {'committed': False}
-
-        def onCommit_(result, info):
-            info['committed'] = result
-            return result
-
-        wrapCommit(self.thingie, onCommit_, data)
+        commits = observeCommits(self.thingie)
```

Comment:
just a comment, but since you have updated the test, maybe you know more about the purpose of this test and add a docstring